test/relayFeeCalculator.test.ts

```diff
@@ -74,6 +75,7 @@ class ExampleQueries implements QueryInterface {
   constructor(private defaultGas = "305572") {}
 
   getGasCosts(): Promise<TransactionCostEstimate> {
+    console.log("#####");
```

Comment:
fwiw you can use the logger in test by running `LOG_IN_TEST=yes yarn test ...`.

Reply:
ah ok good to know